src/js/components/CreateServiceModalServicePickerOption.js

```diff
@@ -0,0 +1,48 @@
+import {Hooks} from 'PluginSDK';
+import React from 'react';
+
+import CreateServiceModalServicePickerOptionContent from './CreateServiceModalServicePickerOptionContent';
+import CreateServiceModalServicePickerOptionImage from './CreateServiceModalServicePickerOptionImage';
+import Panel from './Panel';
+
+function getChildOfType(children, type) {
+  return React.Children.toArray(children).filter(
+    (child) => child.type === type
+  );
+}
+
+function CreateServiceModalServicePickerOption({children, onOptionSelect}) {
+  const columnClass = Hooks.applyFilter(
```

Comment:
Why do we need to make these classes customizable. Let's please try to use hooks and mounts only when absolutely necessary.